### PR TITLE
MenuItem tweaks

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] minheight / padding on items
- [x] padding around divider

@keybase/react-hackers @cecileboucheron

### CODE REVIEW

**Comment by cecileboucheron** [comment]
We also need 8px paddings above the list and below the Close.

**Comment by chrisnojima** [comment]
updated

File: shared/common-adapters/floating-menu/menu-layout/index.native.js [hunk 1]
```diff
@@ -89,6 +89,8 @@ const styles = Styles.styleSheetCreate({
     alignItems: 'stretch',
     backgroundColor: Styles.globalColors.white,
     justifyContent: 'flex-end',
+    paddingBottom: Styles.globalMargins.tiny,
+    paddingTop: Styles.globalMargins.tiny,
```
Comment (cecileboucheron) [inline shared/common-adapters/floating-menu/menu-layout/index.native.js]: Let's do `small` here instead of tiny - 8px still looks tight.

**Comment by chrisnojima** [comment]
updated

**Pull request merged**